sdks/python/apache_beam/runners/worker/statesampler.py

```diff
@@ -48,6 +48,12 @@ def get_current_tracker():
     return None
 
 
+def for_test():
+  from apache_beam.utils.counters import CounterFactory
```

Comment:
Can we move this to the top?  There don't seem to be any circular dependency issues.